src/main/java/com/customerApi/api/services/CustomerDeleteService.java

```diff
@@ -0,0 +1,73 @@
+package com.customerApi.api.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.stereotype.Service;
+
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.repositories.CustomerRepository;
+
+
+@Service
+public class CustomerDeleteService {
+	
+	public static final Logger logger=LoggerFactory.getLogger(CustomerDeleteService.class) ;
+
+	@Autowired
+	CustomerRepository repo ;
+	
+	
+	
+	public boolean deleteCustomerById(int id)
+	{
+		Customer customer = new Customer();
+		boolean status = false;
+			
+		logger.info("Inside delete service:"+customer);
+		try {
+			if(id!=0)
+			{
+				
+				repo.deleteById(id);			
+				logger.info("Customer with id: "+id+" has been deleted successfully");
+				status= true;
+			}
+			else
+			{
+				logger.info("Enter valid cutomer id");
+				status= false;
+			}
+		}catch(NullPointerException e) {
+			e.printStackTrace();
+		}
+		
+		
+		return status;
+	}
+	
+	public boolean deleteCustomer(Customer record)
+	{
+		boolean status=false;
+		try {	
+		if(record!=null)
+			{
+				
+				repo.delete(record);;
+				logger.info("All Customer has been deleted successfully");
```

Comment:
The proper way of printing log is below, its an example
_For example, if you want to print 1 value"_
logger.info("Customer Name {}, sign-in in", customer.getName());

_For example, if you want to print 2 values or multiple"_
logger.info("Customer First Name {}, last name {}, sign-in in", customer.getFirstName(), customer.getLastName());